tlf/handle_test.go

```diff
@@ -241,6 +285,30 @@ func TestHandleAccessorsPublic(t *testing.T) {
 		})
 }
 
+func TestHandleAccessorsSingleTeam(t *testing.T) {
+	w := []keybase1.UserOrTeamID{
+		keybase1.MakeTestTeamID(4).AsUserOrTeam(),
+	}
+
+	h, err := MakeHandle(
+		w, nil, nil, nil, nil)
+	require.NoError(t, err)
+
+	require.Equal(t, SingleTeam, h.Type())
+
+	// All types of IsWriter/IsReader calls should panic for team TLFs.
```

Comment:
Yeah, my tentative plan is that `tlf/handle.go` would continue to panic, to avoid having to expand that package to import contexts and deal with potentially blocking RPC calls.  Maybe that's a bad call, I don't really know, but I think it was initially designed to be as small as possible.